addons/Connection/src/presenter.js

```diff
@@ -1017,6 +1028,95 @@ function AddonConnection_create() {
         }
     };
 
+    function isAnswersEqualToCorrectResults (userAnswers, correctAnswers) {
+        userAnswers.forEach(function(element, index) {
+            this[index] = element.join('');
+        }, userAnswers);
+        correctAnswers.forEach(function(element, index) {
+            this[index] = element.join('');
+        }, correctAnswers);
+
+        if (userAnswers === correctAnswers) return true;
```

Comment:
To chyba nigdy nie będzie spełnione? 
Vide:
[1] === [1]; // false

Reply:
zgadza się, usunąłem linijkę